tox.ini

```diff
@@ -0,0 +1,77 @@
+[tox]
```

Comment:
You should probably return tox.ini to whatever came from the cookiecutter and just take these changes:
https://github.com/edx/cookiecutter-django-app/pull/28/files